tests/gem5/asmtest/configs/riscv_asmtest.py

```diff
@@ -170,8 +88,8 @@
 )
 
 # Set the workload
-binary = Resource(args.resource, resource_directory=args.resource_directory)
-motherboard.set_se_binary_workload(binary, arguments=args.arguments)
+binary = obtain_resource(args.resource)
```

Comment:
So, I'd quite like there to continue to be an `resource_directory=args.resource_directory` option here. When testing I store resources downloaded in "tests/gem5/resources".